pkg/operator/ceph/controller/spec.go

```diff
@@ -798,3 +800,9 @@ func ConfigureExternalMetricsEndpoint(ctx *clusterd.Context, monitoringSpec ceph
 func extractMgrIP(rawActiveAddr string) string {
 	return strings.Split(rawActiveAddr, ":")[0]
 }
+
+func GetContainerImagePullPolicy(containerImagePullPolicy string, opConfig OperatorConfig) *v1.PullPolicy {
```

Comment:
How about a unit test around this function and its defaults?

Reply:
Done